src/netaddress.cpp

```diff
@@ -429,20 +626,20 @@ uint32_t CNetAddr::GetLinkedIPv4() const
 }
 
 uint32_t CNetAddr::GetNetClass() const {
-    uint32_t net_class = NET_IPV6;
-    if (IsLocal()) {
-        net_class = 255;
-    }
+    // Make sure that if we return NET_IPV6, then IsIPv6() is true. The callers expect that.
```

Comment:
8d2b60cb nit, clang format while touching this
```diff
-uint32_t CNetAddr::GetNetClass() const {
+uint32_t CNetAddr::GetNetClass() const
+{
```

Reply:
Done.